src/com/cloudogu/ces/cesbuildlib/Git.groovy

```diff
@@ -67,6 +67,34 @@ class Git implements Serializable {
         return branchName.substring(branchName.lastIndexOf('/') + 1)
     }
 
+    /**
+     * @return if this branch is a release branch according to git flow
+     */
+    boolean isReleaseBranch() {
+        return getBranchName().startsWith("release/")
+    }
+
+    /**
+     * @return true if this branch differs from the develop branch
+     */
+    boolean developHasChanged(String branchName){
```

Comment:
Please make more versatile, eg like so `branchesHaveDiverged(String source, target)`.
This facilitates reuse.